ui/app/templates/application.hbs

```diff
@@ -1,2 +1,19 @@
 {{partial "svg-patterns"}}
-{{outlet}}
+{{#unless error}}
+  {{outlet}}
+{{else}}
+  <div class="error-container">
```

Comment:
have you considered extracting this as a partial, and then using in it other places (though it seems like all of the pertinent places are currently covered)?

Reply:
Yeah, I toyed with making it a component so I could just do something like

```hbs
{{#error-container title="Not Found"}}
Some <em>markup friendly</em> message goes here
{{/error-container}}
```

but decided it was premature refactoring.